### DISCUSSION TITLE
No text displayed in NixOS

### DISCUSSION THREAD

**Discussion by themg95** [KRTirtho/spotube#1949] [Q&A]
### Is there an existing issue for this? (Please read the description)

- [X] I have searched the existing issues

### Current Behavior

When launching the app, all the text (at least on the login/first start screen) is blank
![image](https://github.com/user-attachments/assets/f7809a29-4085-4c72-9ea2-cb2bf5a04c45)

### Expected Behavior

There should be text

### Logs

<details>
<summary>Logs</summary>

```
can't access logs
```
</details>

### Operating System

NixOS aarch64

### Spotube version

v3.8.1

### Installation source

GitHub Releases (Binary), WinGet (Windows)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
You're probably missing some fonts. Try installing the microsoft fonts.
It's a OS orr Flutter issue, not related to Spotube
